### ISSUE TITLE
 OpenCV::DNN doesn't support conv1d 

### ISSUE THREAD

**Issue by azuredsky** [opencv/opencv#18205]
conv = nn.Conv1d(1, 1, kernel_size=k_size, padding=(k_size - 1) // 2, bias=False) does not spport?

opencv: 4.4.0
os:centos 7
python: 3.7

net = dnn.readNetFromONNX('model_best.onnx')
cv2.error: OpenCV(4.4.0) /tmp/pip-req-build-n0_oeb8j/opencv/modules/dnn/src/onnx/onnx_importer.cpp:209: error: (-215:Assertion failed) attribute_proto.ints_size() == 2 || attribute_proto.ints_size() == 3 in function 'getLayerParams'

**Comment by berak** [comment]
i would also like to see this (feature request, hey ;) as it would allow networks like https://github.com/eeskimez/noise_resilient_3dtface to be used with opencv

(generating facial landmarks from audio)

**Comment by asmorkalov** [comment]
@azuredsky could you provide the model with dummy weights to reproduce the issue?

**Comment by berak** [comment]
@asmorkalov 

it can be reprocuced with:

```
import torch, torch.nn as nn
import cv2

net = nn.Conv1d(1, 1, kernel_size=3, padding=(2) // 2, bias=False)
input = torch.randn(1,1,512)
torch.onnx.export(net, input, "conv1d.onnx", verbose=True)
cv2.dnn.readNet("conv1d.onnx")
```

same error (self built on colab)
```
error: OpenCV(4.4.0-dev) /content/opencv/modules/dnn/src/onnx/onnx_importer.cpp:209: error: (-215:Assertion failed) attribute_proto.ints_size() == 2 || attribute_proto.ints_size() == 3 in function 'getLayerParams'
```

[conv1d.onnx.txt](https://github.com/opencv/opencv/files/5209132/conv1d.onnx.txt) (remove .txt extension)

**Comment by asmorkalov** [comment]
Thanks a lot!
